java/Ice/greeter-android/app/src/main/java/com/example/ice/androidgreeter/App.kt

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.example.ice.androidgreeter
```

Comment:
It would be nice for the demo name and package name to be consistent: either greeter android or android greeter, not demo=greeter-android and package=androidgreeter.

I prefer "Android Greeter".